src/test/resources/log4j2-test.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+    <Appenders>
+        <Console name="STDOUT" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d %-5p [%t] %C{2} (%F:%L) - %m%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <Root level="WARN">
+            <AppenderRef ref="STDOUT"/>
+        </Root>
+        <Logger name="com.amazonaws.codepipeline" level="INFO"/>
```

Comment:
Default level for log4j was debug, do we want to set the same for log4j2?

Reply:
this was intentional, debug is too noisy